subprojects/core/src/main/java/org/gradle/plugin/repository/PluginRepositoriesSpec.java

```diff
@@ -24,6 +24,10 @@
  * <p>
  * Plugin repositories added via this interface will used to resolve plugins specified
  * in the <code>plugins {}</code> block.
+ *
+ * When defining multiple <code>pluginRepository {}</code> blocks (like possible with
```

Comment:
I don't think this clarification is necessary, because this is true for any domain object container in Gradle